src/components/bridge/BridgeSelection.vue

```diff
@@ -185,20 +218,25 @@ import { useAccount, useNetworkInfo } from 'src/hooks';
 import { EthBridgeNetworkName } from 'src/modules/zk-evm-bridge';
 import { Path as RoutePath, buildEthereumBridgePageLink } from 'src/router/routes';
 import { computed, defineComponent } from 'vue';
-import { stargateUrl, layerSwapLink, relayBridgeAppLink } from 'src/modules/zk-evm-bridge/index';
+import {
+  stargateUrl,
+  layerSwapLink,
+  relayBridgeAppLink,
+  zkatanaBridgeUrl,
```

Comment:
zKatana

Reply:
fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1238/commits/73807824eb5f82f09b41027309290152d704db13) commit